libs/content/workspace-ffi/SwiftWorkspace/Sources/Editor/iOSMTK.swift

```diff
@@ -11,12 +11,12 @@ import GameController
 public class iOSMTKTextInputWrapper: UIView, UITextInput, UIDropInteractionDelegate {
```

Comment:
While we're cleaning things up can we eliminate the use of `Editor` in this file path?